auditbeat/module/file_integrity/event.go

```diff
@@ -354,6 +382,16 @@ func buildMetricbeatEvent(e *Event, existedBefore bool) mb.Event {
 		}
 	}
 
+	if e.Process != nil {
+		file["process.entity_id"] = e.Process.EntityID
+		file["process.name"] = e.Process.Name
+		file["process.pid"] = e.Process.PID
+		file["process.user.id"] = e.Process.User.ID
+		file["process.user.name"] = e.Process.User.Name
+		file["process.group.id"] = e.Process.Group.ID
+		file["process.group.name"] = e.Process.Group.Name
```

Comment:
This will produce JSON keys that contain dots. We like our events to use nested objects because all of the current processing in Beats and Ingest Node assumes key names do not contain dots. My recommendation is to form this into a map like `file["process"] = mapstr.M{"entity_id": e.Process.EntityID, ...}`.